rapstore-frontend/src/app/app-build/app-build.component.ts

```diff
@@ -66,6 +75,13 @@ export class AppBuildComponent implements OnInit {
      
   }
 
+  show_error_toast(msg: string, reason: string) {
+      this.notificationsService.error(msg, reason, {
+        timeOut: 5000,
```

Comment:
Error message should not disappear automatically, e.g. in case the user is not using his machine when build has failed.